compiler/luci/service/src/Nodes/CircleReshape.cpp

```diff
@@ -138,8 +138,14 @@ loco::TensorShape Algorithm::visit(const luci::CircleReshape *node)
   uint32_t input_element_count = 1;
   uint32_t output_element_count = 1;
   uint32_t unknown_dim_index = UINT32_MAX;
+  bool should_infer = true;
```

Comment:
maybe use `is_static_shape` ?

Reply:
Sounds great! I've changed it into `is_static_shape`.